src/main/java/org/gridsuite/modification/server/repositories/NetworkModificationRepository.java

```diff
@@ -99,13 +102,22 @@ public void saveModifications(UUID groupUuid, List<ModificationEntity> modificat
                 .collect(Collectors.toList());
     }
 
-    public ElementaryModificationInfos getElementaryModification(UUID groupUuid, UUID modificationUuid) {
+    public ElementaryAttributeModificationInfos getElementaryModification(UUID groupUuid, UUID modificationUuid) {
         return ((ElementaryModificationEntity<?>) this.modificationRepository
                 .findById(modificationUuid)
                 .filter(m -> ModificationType.ELEMENTARY.name().equals(m.getType()))
                 .filter(m -> groupUuid.equals(m.getGroup().getId()))
                 .orElseThrow(() -> new NetworkModificationException(MODIFICATION_NOT_FOUND, modificationUuid.toString())))
-                .toElementaryModificationInfos();
+                .toElementaryAttributeModificationInfos();
+    }
+
+    public LoadCreationInfos getLoadCreationModification(UUID groupUuid, UUID modificationUuid) {
+        return ((CreateLoadEntity) this.modificationRepository
+            .findById(modificationUuid)
+            .filter(m -> ModificationType.LOAD_CREATION.name().equals(m.getType()))
```

Comment:
Type et groupUUId not necessary ?

Reply:
Both methods getEquipmentAttributeModification and getLoadCreationModification are used only in tests
filtering with the type ensure the cast to the right entity
filtering with the groupUuid verifies that the modification is in the right group